.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build APK
```

Comment:
Why do we need this script? Let's not pile up more changes please. If you really want this part, please create a separate pull request and we can discuss it there.